sympy/physics/matrices.py

```diff
@@ -153,25 +154,18 @@ def mgamma(mu, lower=False):
     (0, 0, 0, -1)
 ))
 
+
+@deprecated(issue=20246, useinstead="DFT(n).as_mutable(), DFT(n), DFT(n).as_explicit()",
+            deprecated_since_version="1.7")
 def mdft(n):
     r"""
-    Returns an expression of a discrete Fourier transform as a matrix multiplication.
-    It is an n X n matrix.
-
-    References
-    ==========
+    Deprecated. Use DFT from sympy.matrices.expressions.fourier instead.
 
-    .. [1] https://en.wikipedia.org/wiki/DFT_matrix
-
-    Examples
+    See Also
     ========
 
-    >>> from sympy.physics.matrices import mdft
-    >>> mdft(3)
-    Matrix([
-    [sqrt(3)/3,                sqrt(3)/3,                sqrt(3)/3],
-    [sqrt(3)/3, sqrt(3)*exp(-2*I*pi/3)/3,  sqrt(3)*exp(2*I*pi/3)/3],
-    [sqrt(3)/3,  sqrt(3)*exp(2*I*pi/3)/3, sqrt(3)*exp(-2*I*pi/3)/3]])
+    sympy.matrices.expressions.fourier
```

Comment:
This needs some fix for the broken reference

Reply:
Yes, I realized that I (probably) have to create that part of the documentation in the corresponding rst file. May be that I simply remove it as I unfortunately have limited understand of that part.

Reply:
Yes I neither can anticipate any consequences for applying any suggestions, unless I myself test out the build

Reply:
I removed the "See Also" part and improved the deprecation replacement documentation a bit.